includes/functions/llms-functions-templates-shared.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Shared template functions
+ *
+ * A "shared" function is any function used by more than one post type.
+ *
+ * @package LifterLMS/Functions
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+if ( ! function_exists( 'llms_template_instructors' ) ) {
+
+	/**
+	 * Get single post instructors template
+	 *
+	 * Used by courses and membership.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	function llms_template_instructors() {
+
+		$post = llms_get_post( get_the_ID() );
```

Comment:
```suggestion
		$llms_post = llms_get_post( get_the_ID() );
```
I would love to avoid any kind of confusion with the global `$post`, what do you think?